README.md

```diff
@@ -48,6 +48,17 @@ are identical and can't get mangled by a broken deps installation.
 
 We're using webpack to build a JS bundle.
 
+### Hot module reloading
+
+If you run the webpack-dev-server you can get hot module reloading.
+
+For example run `grunt serve`
+
+And then visit http://localhost:8080/webpack-dev-server/management.html
```

Comment:
Hmm, yeah - will need to think about that. It'll probably need Charles Proxy or maybe we can do something with nginx?